test/parser_test.go

```diff
@@ -0,0 +1,363 @@
+package test
```

Comment:
move this file to be in the ini package with the name `parser_test.go` in go test files exist directly next to the code files